src/NavAbilitySDK.jl

```diff
@@ -6,13 +6,21 @@ include("./archive/NavAbilitySDK.jl")
 @reexport using .ArchivedNavAbilitySDK
 
 import .ArchivedNavAbilitySDK.getVariable # Don't want to override with new implementations
+import .ArchivedNavAbilitySDK.getVariables # Don't want to override with new implementations
+import .ArchivedNavAbilitySDK.listVariables # Don't want to override with new implementations
 import .ArchivedNavAbilitySDK.ls # Don't want to override with new implementations
 import .ArchivedNavAbilitySDK.getFactor # Don't want to override with new implementations
-import .ArchivedNavAbilitySDK.lsf # Don't want to override with new implementations
 import .ArchivedNavAbilitySDK.getFactors # Don't want to override with new implementations
+import .ArchivedNavAbilitySDK.listFactors # Don't want to override with new implementations
+import .ArchivedNavAbilitySDK.lsf # Don't want to override with new implementations
+
+# Graphql
+include("./navability/graphql/Factor.jl")
+include("./navability/graphql/Variable.jl")
+export GQL_FRAGMENT_FACTORS, GQL_GETFACTOR, GQL_GETFACTORS, GQL_GETFACTORSFILTERED
```

Comment:
Think we're only going to use these in the project, right? In which case we don't need to export them.